Rakefile

```diff
@@ -4,5 +4,20 @@ require 'middleman-gh-pages'
 require 'rspec/core/rake_task'
 RSpec::Core::RakeTask.new(:spec)
 
+desc "Compile javascript -> coffeescript"
+task :coffee do
+  destination = "source/javascripts/refills/coffeescript"
+  Dir["source/javascripts/refills/**/*.js"].each do |source|
+    filename = File.basename(source.gsub(%r{.js}, ".coffee"))
+
+    `js2coffee #{source} > #{destination}/#{filename}`
```

Comment:
I don't know enough how `js2coffee` works and which errors it might spit out to guard against them. Is it really necessary?